### ISSUE TITLE
Skiplinks problema di accessibilità AA 1.4.3 Contrast (Minimum) - Color contrast is insufficient

### ISSUE THREAD

**Issue by riccafi** [italia/bootstrap-italia#523]
## Comportamento atteso
Gli skiplinks dovrebbero avere un contrasto sufficiente fra testo e background.

## Comportamento attuale
Il contrasto fra testo e background non è sufficiente. Un contrasto sufficiente dovrebbe avere un rapporto di contrasto fra testo e background almeno di 4.5, adesso è di 2.93 .

## Possibili soluzioni
Aumentare il contrasto di default fra testo e background così che superi la soglia di rapporto di contrasto di 4.5 .

## Contesto
Per riprodurre la casistica è sufficiente accedere alla sezione skiplinks di bootstrap italia e lanciare un controllo con tool comuni di analisi di accessibilità (ad esempio SiteImprove).
